src/lexer_test.cc

```diff
@@ -89,10 +89,10 @@ TEST(Lexer, CommentEOF) {
 
 TEST(Lexer, Tabs) {
   // Verify we print a useful error on a disallowed character.
-  Lexer lexer("   \tfoobar");
+  Lexer lexer("\tcommand = foo $in");
   Lexer::Token token = lexer.ReadToken();
   EXPECT_EQ(Lexer::INDENT, token);
+
   token = lexer.ReadToken();
-  EXPECT_EQ(Lexer::ERROR, token);
-  EXPECT_EQ("tabs are not allowed, use spaces", lexer.DescribeLastError());
```

Comment:
Please add tests to also verify that:

- line continuations with tabs actually work. E.g.:

```
build foo: rule_name $
<tab>input1 $
<tab>input2
```
- Tabs that appear after indentation are *still* not allowed, except when they are in comments. E.g.

rule correct_rule
<tab>command = foo bar

rule incorrect_rule
<tab>command = foo<tab>bar

build incorrect_build: rule_name<tab>input1<tab>input2
```

The most important reason to avoid tabs is to avoid using them in `command = ` definitions because this will be equally confusing to users.

Finally, please update manual.doc appropriately to document the new behavior.

Reply:
Thank you, I have indeed tweak the indentation after a linebreak.

However,  even without my changes ` command = foo<tab>bar` is fine to use.

Reply:
Ack, then no need to test for this then. Thank you for double-checking.